packages/cli/package.json

```diff
@@ -56,11 +55,11 @@
   "homepage": "https://github.com/posva-sponsors/vue-termui#readme",
   "dependencies": {
     "picocolors": "^1.0.0",
-    "vite-node": "^0.10.0",
+    "vite-node": "^0.22.1",
     "ws": "^8.5.0"
   },
   "peerDependencies": {
-    "vite": "^2.8.6",
+    "vite": "^2.0.0 || ^3.0.0-0",
```

Comment:
Do you think we will be able to support both versions?

Reply:
I prefer to support Vite3 only.